src/services/__tests__/helpers.spec.tsx

```diff
@@ -18,79 +18,108 @@ describe('Calculation of next exercise', () => {
     numberOfChildren: 2,
     isLeaf: false,
     parentTitle: null,
-    needsTrainingSetEndpoint: false
+    needsTrainingSetEndpoint: false,
+    leafDisciplines: [1, 2]
   }
 
-  const getNextExerciseWithCheck = async (): Promise<NextExercise> => {
-    const e = await getNextExercise(profession)
-    if (!e) {
-      throw Error('NextExerciseData is undefined')
+  describe('Calculation of next exercise', () => {
```

Comment:
Our (undocumented) convention is to always name describe blocks the same as the component/function we are testing.
```suggestion
  describe('getNextExercise', () => {
```